dts/bindings/wifi/espressif,esp8266.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2019 Tobias Svehagen
+# SPDX-License-Identifier: Apache-2.0
+
+title: Espressif ESP8266
```

Comment:
Title was deprecated just today.

You'll need to move the Title into the description eg:
Espressif ESP8266  WiFi modem (AT Commands)